src/quartz_api/cmd/server.conf

```diff
@@ -22,6 +22,10 @@ backend {
     database_url = ${?DB_URL}
   }
   dummydb {}
+  dataplatform {
+    host = ${?DATA_PLATFORM_HOST}
+    port = ${?DATA_PLATFORM_PORT}
```

Comment:
Does this match the env vars you put into the other data-platform backed stuff? Just thinking about consistency